### PR TITLE
MEN-5149 Basic service and health check.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@alfrunes this is devicemonitor-based, just health check.

File: main.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+			},
+			{
+				Name:   "migrate",
+				Usage:  "Run the migrations",
+				Action: cmdMigrate,
+			},
+		},
+	}
+	app.Usage = "Device Connect"
+	app.Version = "1.0.0"
```
Comment (lluiscampos) [inline main.go:68]: We should stop hardcoding the version to `1.0.0`. This is only adding confusion. IMO we should just omit this line.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+		-p acceptance down
+
+
+.PHONY: fmt
+fmt:
+	$(GOFMT) -s -w $(GOFILES)
+
+.PHONY: lint
+lint:
+	golangci-lint run -v
```
Comment (alfrunes) [inline Makefile:109]: There's no `.golangci.yml` for this repo.

File: main.go (hunk 1 above)
Comment (merlin-northern) [reply main.go:68]: I have no idea, nad no problem with omitting this line, the problem is, we are doing it for others repos. how about keeping it "for now" and discussing tomorrow during BET meeting? (the discussion about versioning of backend services is long over due), makes sense?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+version: "3"
```
[the remark this answers is not in the document]
Comment (merlin-northern) [reply docker-compose.yml]: are you asking me?
wow. sorry about that, some left over. removed.

File: main.go (hunk 1 above)
Comment (lluiscampos) [reply main.go:68]: We discussed it over PRs that we had some months ago. My suggestion is to remove it from here to stop propagating it to new repos.

Comment (merlin-northern) [reply main.go]: removed.
yes, and I asked back then if we can discuss removing versioning of backend services. :)

**Pull request merged**